domains/matrixcoder.json

```diff
@@ -4,11 +4,11 @@
         "email": "tkgoutamm625@gmail.com"
     },
     "record": {
-        "A": ["217.174.245.249"],
+        "CNAME": "matrixcoder0101.github.io"
```

Comment:
Cannot have CNAME in conjunction with other records.

Reply:
> Cannot have CNAME in conjunction with other records.

Sir any other way to use this domain in both GitHub Pages and ImproveMX email server ?

Reply:
> > Cannot have CNAME in conjunction with other records.
> 
> Sir any other way to use this domain in both GitHub Pages and ImproveMX email server ?

I believe GitHub allows you to do A records, you would have search in the repository settings. And then add your ImproveMX email settings which I assume are TXT and MX records normally. CNAME records just don't allow anything else.

Reply:
Now check Is this are record correct?